dask_geopandas/array.py

```diff
@@ -0,0 +1,50 @@
+import dask
+from dask.utils import M, OperatorMethodMixin
+import dask.array as da
+from dask.array.utils import meta_from_array
+
+import geopandas
+from geopandas.array import _points_from_xy
+
+
+def points_from_xy(x, y, z=None):
+    """Convert arrays of x and y values to a GeometryArray of points."""
+    x = da.asarray(x)
+    y = da.asarray(y)
+    if z is not None:
+        z = da.asarray(z)
+    out = _points_from_xy(x, y, z)
+    aout = da.empty(len(x))
+    aout[:] = out
+    return GeometryArray.from_array(aout)
+
+
+class GeometryArray(da.core.Array, OperatorMethodMixin):
```

Comment:
Just wondering: what is the reason that you also need to create a dask version of `GeometryArray` ?

My assumption was that only GeoSeries/GeoDataFrame would be needed, since those would be consisting of (geo)pandas chunks that use geopandas' GeometryArray extension array.

Reply:
Yeah, I think it's not actually needed. I just kind of stubbed it out in the first pass (I should mention that this branch is very WIP). I originally thought that points_from_xy would be a good first entrypoint to support.

Reply:
Yeah, I think `points_from_xy` *is* a good entrypoint, but probably can also return a GeoSeries directly to avoid needing a partitioned array.

Reply:
Ok good idea :)